roofit/roofitcore/src/RooPlot.cxx

```diff
@@ -377,13 +345,10 @@ RooPlot::~RooPlot()
 {
   // Delete the items in our container and our iterator.
   if (_dir) {
-    if (!_dir->TestBit(TDirectoryFile::kCloseDirectory)) {
-      _dir->GetList()->RecursiveRemove(this) ;
-    }
+    _dir->GetList()->RecursiveRemove(this) ;
```

Comment:
it seems that this change belongs to the second commit but ended up in the first

Reply:
Could also be put in the second, true.